src/test/groovy/edu/ie3/datamodel/models/CommonVoltageLevelTest.groovy

```diff
@@ -48,13 +48,4 @@ class CommonVoltageLevelTest extends Specification {
 		expect:
 		!dut.covers("HS", Quantities.getQuantity(110d, KILOVOLT))   // May be false
 	}
-
```

Comment:
Please alter this test, as not the function has gone, but become a different output. Please test for the return of `false`.

Reply:
The way you resolve this, obviously depends on the outcome of the discussion of the issue.

Reply:
Is this still valid @ckittl? I think this can be resolved can't it?